core/Command/Background/ListCommand.php

```diff
@@ -67,7 +67,11 @@ protected function configure(): void {
 
 	protected function execute(InputInterface $input, OutputInterface $output): int {
 		$jobs = $this->jobList->getJobsIterator($input->getOption('class'), (int)$input->getOption('limit'), (int)$input->getOption('offset'));
-		$this->writeTableInOutputFormat($input, $output, $this->formatJobs($jobs));
+		$jobsInfo = $this->formatJobs($jobs);
+		$this->writeTableInOutputFormat($input, $output, $jobsInfo);
+		if ($input->getOption('output') === self::OUTPUT_FORMAT_PLAIN && count($jobsInfo) >= (int)$input->getOption('limit')) {
+			$output->writeln("\n<comment>Output is currently limited to first " .  $input->getOption('limit') . " jobs. Specify `-l, --limit[=LIMIT]` to override.</comment>");
```

Comment:
Depending on the offset it's not "first" 😅


```suggestion
			$output->writeln("\n<comment>Output is currently limited to " .  $input->getOption('limit') . " jobs. Specify `-l, --limit[=LIMIT]` to override.</comment>");
```